app/assets/themes/fearlesscities/_navbar.scss

```diff
@@ -0,0 +1,45 @@
+@import "variables";
+
+.title-bar {
+  a {
+    font-family: "Poppins Bold";
+    color: $fearless-blue;
+  }
+}
+
+.navbar {
+  background-color: white;
+  padding-bottom: 1em;
+
+  .main-nav ul {
+    text-transform: uppercase;
+
+    .main-nav__link {
+      a {
+        color: $fearless-blue;
+        box-shadow: none;
+
+        &:hover {
+          box-shadow: none;
+          text-decoration: underline;
+        }
+      }
+
+      &.main-nav__link--active a {
+        box-shadow: none;
+        border: 1px solid $fearless-black;
+        border-radius: 10em;
```

Comment:
Could this be applied to all buttons?

Reply:
This ```boder-radius``` is used only on those buttons that are set to "active", you can see that only here i also define the ```border``` and the ```background-color```. I thought this was the correct place to set it, but maybe there is a better one?